package-lock.json

```diff
@@ -1,8 +1,2853 @@
 {
   "name": "one-vscode",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
why is this changed?

Reply:
[lockfileVersion]
1: The lockfile version used by npm v5 and v6.
2: The lockfile version used by npm v7, which is backwards compatible to v1 lockfiles.

What version of npm are you using? I'm using v7.12.1

Reply:
```
$ npm --version
6.14.10
```

Reply:
Thanks for checking😀
Even if npm is version 6, `lockfileversion` 2 is [compatible](https://stackoverflow.com/questions/66150813/is-lockfileversion-2-in-package-lock-json-from-npm-7-compatible-with-older-ve).
`lockfileversion` is an option that is automatically converted when using npm v7.
How about using `lockfileversion` to 2?

Reply:
As I'm not used to `npm` domain, I don't know it's OK or not.
I'll just follow your words.